machine/install-machine.md

```diff
@@ -37,15 +36,20 @@ docker/machine release page</a> on GitHub.
     sudo cp /tmp/docker-machine /usr/local/bin/docker-machine
     ```
 
-    If you are running with **Windows** with git bash:
+    If you are running with **Windows** with [Git BASH](https://git-for-windows.github.io/){: target="_blank" class="_"}:
 
     ```console
     $ if [[ ! -d "$HOME/bin" ]]; then mkdir -p "$HOME/bin"; fi && \
 curl -L https://github.com/docker/machine/releases/download/v0.12.2/docker-machine-Windows-x86_64.exe > "$HOME/bin/docker-machine.exe" && \
 chmod +x "$HOME/bin/docker-machine.exe"
     ```
 
-    Otherwise, download one of the releases from the <a href="https://github.com/docker/machine/releases/" target="_blank"> docker/machine release page</a> directly.
+    > The above command will work on Windows only if you use a
```

Comment:
I guess I'm not sure what you mean re: the associated one-liners with every [Machine release](https://github.com/docker/machine/releases/), and I don't see [looking at the docs](https://docs.docker.com/machine/install-machine/#installing-machine-directly) what we are doing differently here for Windows vs. Mac? (other than the clarification on using Git Bash)